src/migration-scripts/interfaces/16-to-17

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Command line migration of port mirroring
+# https://phabricator.vyos.net/T3089
+
+import sys
+from vyos.configtree import ConfigTree
+
+if __name__ == '__main__':
+    if (len(sys.argv) < 1):
+        print("Must specify file name!")
+        sys.exit(1)
+
+    file_name = sys.argv[1]
+
+    with open(file_name, 'r') as f:
+        config_file = f.read()
+
+    config = ConfigTree(config_file)
+    base = ['interfaces', 'ethernet']
+    if not config.exists(base):
+        # Nothing to do
+        sys.exit(0)
+    else:
+        interfaces_list = []
+        mirror_base = []
+        for interface in config.list_nodes(base):
+            mirror_old_base = base + [interface, 'mirror']
+            if config.exists(mirror_old_base):
+                for mirror_int in config.list_nodes(mirror_old_base):
```

Comment:
There can only be one mirror on the current implementation, please not make the migrator harder then it is.
Please see the code from the migrator I shared in this PR.